jvm/build.gradle

```diff
@@ -4,18 +4,12 @@
 
 
 buildscript {
-    ext.kotlin_version = '1.1.0'
-
     repositories {
         mavenCentral()
     }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
 }
 
-apply plugin: 'kotlin'
```

Comment:
I don't know the gradle internals, but the Kotlin build scripts know about and invoke their Java equivalents. A Kotlin build entails a Java build. (Scala works the same way, and I'll bet $5 Clojure does, too.)